<launch>
    <node pkg="dnetwork"
          name="dnetwork_4"
          type="dnetwork_node"
          output="screen"
          respawn="false"
          required="true"
          cwd="node"
          args="">

        <env name="ROSCONSOLE_CONFIG_FILE" value="$(find dlaunch)/drosconsole.conf"/>

        <param name="RobotId" value="4" />

    </node>

    <rosparam command="load" file="$(find dconfig)/global.yml"/>

</launch>
